lib/src/connect/meeting/main_websocket/user/user.dart

```diff
@@ -17,7 +17,6 @@ class UserModule extends Module {
 
   @override
   void onConnected() {
-    subscribe("current-user");
```

Comment:
Braucht man doch nicht?

Reply:
Doch, aber ich habe das nun ins Mainwebsocket eingebaut, weil im Web Client ist folgender Ablauf:

Validate User
Warten auf Confirm
Sub to current-user
Validate User nochmal